### ISSUE TITLE
Cuda not compatible with PyTorch installation error while training Blender with 8xA100

### ISSUE THREAD

**Issue by kazeto111** [facebookresearch/ParlAI#3943]
I implemented `python parlai/scripts/train_model.py -t blended_skill_talk --model transformer/generator --model_file outputs/models/model --init_model zoo:blender/blender_3B/model --dict_file zoo:blender/blender_3B/model.dict --datapath outputs/data --download_path outputs/data --n_heads 4 --n_encoder_layers 2 --n_decoder_layers 24 --n_positions 128 --delimiter --label_truncate 128 --ffn_size 10240 --embedding_size 2560 --activation gelu --variant prelayernorm --dropout 0.1 --special_tok_lst [U1],[U2] --dict_lower true --dict_tokenizer bytelevelbpe --optimizer adafactor --warmup_updates 100 --batchsize 4 --save_after_valid true --text_truncate 128 --truncate 128 --max_train_time 600 --validation_metric ppl --validation_every_n_epochs 0.25 --fp16 true --fp16_impl mem_efficient --force-fp16-tokens true --update_freq 4 --gradient_clip 0.1 --skip_generation false --dynamic_batching full --model-parallel true
`
this command worked well with 2 x v100, but with 8 x A100, I got the error like below.  (showing num words = 8008 to dedicate the error place)

```
2021-08-16 04:02:29,512 INFO     | num words = 8008
/azureml-envs/azureml_9f42dddb00266f3582208ef8cdab4701/lib/python3.7/site-packages/torch/cuda/__init__.py:104: UserWarning: 
A100-SXM4-40GB with CUDA capability sm_80 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_70.
If you want to use the A100-SXM4-40GB GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(incompatible_device_warn.format(device_name, capability, " ".join(arch_list), device_name))
```

so I visited https://pytorch.org/get-started/locally/ and followed to implement `conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch` but it doesn't work. Neither did `conda install pytorch torchvision torchaudio cudatoolkit=11.1 -c pytorch -c nvidia ` (attached the picture of this site)
<img width="839" alt="スクリーンショット 2021-08-16 14 34 41" src="https://user-images.githubusercontent.com/54926247/129515848-57d1fa7b-d5a1-4838-adbb-7c75fa62f250.png">
<img width="894" alt="スクリーンショット 2021-08-16 14 34 46" src="https://user-images.githubusercontent.com/54926247/129515854-6f345c13-c822-43d9-9789-1b95b33cd4ef.png">

How can I solve this problem?

Thank you so much

**Comment by jytime** [comment]
Pytorch 1.9 does not support A100 by now. You can use 1.8.1+cu111

**Comment by kazeto111** [comment]
Thank you so much!!

I tried but conflict has occurred... I think I should use v100 for now.  Thank you anyway!
